lib/bloc/fee_options/fee_options_bloc.dart

```diff
@@ -34,39 +38,57 @@ class FeeOptionsBloc extends Cubit<FeeOptionsState> {
   }
 
   Future<List<FeeOption>> _constructFeeOptionList(
-    String toAddress,
-    RecommendedFees recommendedFees,
-  ) async {
+      {required String toAddress, required RecommendedFees recommendedFees, String? swapAddress}) async {
```

Comment:
Add comma after last param for indentation.

Reply:
done